docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -194,11 +194,9 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
           break;
 
         case 'menu_block:main':
-          /** @var \Drupal\Core\Plugin\DefaultPluginManager $filters */
-          $filters = \Drupal::service('plugin.manager.config_filter')->getDefinitions();
-          $collegiate_split = 'config_split:collegiate';
-          // 'collegiate' split either doesn't exist or isn't enabled.
-          if (!isset($filters[$collegiate_split]) || !$filters[$collegiate_split]['status']) {
+          $user = \Drupal::currentUser();
+          // Account for user 1 for local development.
+          if ($user->id() != 1 && !in_array('administrator', $user->getRoles())) {
```

Comment:
For consistency can we switch this to use docroot/modules/custom/uiowa_core/src/Access/UiowaCoreAccess.php

Reply:
https://github.com/uiowa/uiowa/blob/be7a1b55c9cd983e13a0b8d78627688ec2eb68b2/docroot/modules/custom/uiowa_aggregator/uiowa_aggregator.module#L28-L45

Reply:
Sounds good to me.

Reply:
I updated this per the example that @richardbporter supplied.